server/webapp/WEB-INF/rails.new/spec/webpack/models/dashboard/dashboard_spec.js

```diff
@@ -49,13 +52,17 @@ describe("Dashboard", () => {
     });
 
     it("it should filter dashboard provided filter text", () => {
-      const dashboard = Dashboard.fromJSON(dashboardData);
+      const dashboard = new Dashboard();
+      dashboard.initialize(dashboardData);
 
       expect(dashboard.allPipelineNames()).toEqual(['up42']);
 
-      expect(dashboard.filterBy("up").allPipelineNames()).toEqual(['up42']);
-      expect(dashboard.filterBy("42").allPipelineNames()).toEqual(['up42']);
-      expect(dashboard.filterBy("up42").allPipelineNames()).toEqual(['up42']);
+      dashboard.searchText("up");
+      expect(dashboard.allPipelineNames()).toEqual(['up42']);
+      dashboard.searchText("42");
+      expect(dashboard.allPipelineNames()).toEqual(['up42']);
+      dashboard.searchText("up42");
+      expect(dashboard.allPipelineNames()).toEqual(['up42']);
```

Comment:
There's only one pipeline, so removing the filtering logic will make the test pass, always.